kalite/distributed/static/css/distributed/introjs.css

```diff
@@ -0,0 +1,343 @@
+.introjs-overlay {
```

Comment:
These are 3rd party definitions. If we want to override them we should add our own styles elsewhere.